templates/base.html

```diff
@@ -45,104 +46,125 @@
         <link rel="apple-touch-icon" sizes="114x114" href= "{% static "img/icons/perdiem_icon_114.png" %}" />
         <link rel="apple-touch-icon" sizes="144x144" href="{% static "img/icons/perdiem_icon_144.png" %}" />
         {% block extrastyle %}{% endblock %}
+        <style>
+        #home-page {
```

Comment:
Just to confirm, should it look like this:

        <style>
        #home-page {
            position: relative;
            background-size: 50%;
            background-repeat: no-repeat;
            background-position: 50%;
            background-color: black;
        }
        </style>

or


<style>
        #home-page {
            position: relative;
            background-size: 50%;
            background-repeat: no-repeat;
            background-position: 50%;
            background-color: black;
        }
</style>